CMakeLists.txt

```diff
@@ -0,0 +1,571 @@
+  libSBRenc/src/ps_encode.h
+  libSBRenc/src/ps_main.h
+  libSBRenc/src/resampler.h
+  libSBRenc/src/sbr.h
+  libSBRenc/src/sbr_def.h
+  libSBRenc/src/sbr_misc.h
+  libSBRenc/src/sbrenc_freq_sca.h
+  libSBRenc/src/sbrenc_ram.h
+  libSBRenc/src/sbrenc_rom.h
+  libSBRenc/src/ton_corr.h
+  libSBRenc/src/tran_det.h)
+
+set(SYS_SRC
+  libSYS/src/genericStds.cpp
+  libSYS/src/syslib_channelMapDescr.cpp)
+
+set(fdk_aacinclude_HEADERS
+  libSYS/include/machine_type.h
+  libSYS/include/genericStds.h
+  libSYS/include/FDK_audio.h
+  libSYS/include/syslib_channelMapDescr.h
+  libAACenc/include/aacenc_lib.h
+  libAACdec/include/aacdecoder_lib.h)
+
+set(libfdk_aac_SOURCES
+    ${AACDEC_SRC} ${AACENC_SRC}
+    ${ARITHCODING_SRC}
+    ${DRCDEC_SRC}
+    ${MPEGTPDEC_SRC} ${MPEGTPENC_SRC}
+    ${SACDEC_SRC} ${SACENC_SRC}
+    ${SBRDEC_SRC} ${SBRENC_SRC}
+    ${PCMUTILS_SRC} ${FDK_SRC} ${SYS_SRC})
+
+if(WIN32 AND BUILD_SHARED_LIBS)
+  set(libfdk_aac_SOURCES ${libfdk_aac_SOURCES} fdk-aac.def)
+endif()
+
+## Create Library target. Actually fdk-aac is enough, but we also create
+## FDK-AAC::fdk-aac for consistence with config-file package.
+
+add_library(fdk-aac ${libfdk_aac_SOURCES})
+add_library(FDK-AAC::fdk-aac ALIAS fdk-aac)
+
+## Library target configuration
+
+### Library target includes
+target_include_directories(fdk-aac
+  PUBLIC
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/libAACdec/include>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/libAACenc/include>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/libSYS/include>
+  PRIVATE
+    libArithCoding/include
+    libDRCdec/include
+    libSACdec/include
+    libSACenc/include
+    libSBRdec/include
+    libSBRenc/include
+    libMpegTPDec/include
+    libMpegTPEnc/include
+    libFDK/include
+    libPCMutils/include)
+
+### Link math library if required
+target_link_libraries(fdk-aac PRIVATE $<$<BOOL:${HAVE_LIBM}>:m>)
+
+### Set public headers and shared library version. Version info is critical for Unix-like OSes.
+set_target_properties(fdk-aac PROPERTIES
+  PUBLIC_HEADER "${fdk_aacinclude_HEADERS}"
+  VERSION 2.0.1
+  SOVERSION 2)
+
+### Some compiler options from Makefile.am
+if(MSVC)
+  target_compile_options(fdk-aac PRIVATE /EHsc)
+else()
+  target_compile_options(fdk-aac PRIVATE -fno-exceptions -fno-rtti)
+endif()
+
+## Library installation
+
+### Note we export `fdk-aac-targets` to use with config-file package.
+install(TARGETS fdk-aac EXPORT fdk-aac-targets
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT BIN
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT DEV
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT DEV
+  PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/fdk-aac COMPONENT DEV)
+
+### Configure and install pkg-config module.
+if(FDK_AAC_INSTALL_PKGCONFIG_MODULE)
+  set(prefix ${CMAKE_INSTALL_PREFIX})
+  set(exec_prefix "\$\{prefix\}")
+  set(libdir "\$\{prefix\}/${CMAKE_INSTALL_LIBDIR}")
```

Comment:
To closer match what the existing pkg-config file contains, I'd use `exec_prefix` instead of `prefix` in `libdir`.